files/views.py

```
@@ -988,7 +991,28 @@ def get(self, request, format=None):
+                # Get the scheduled featured video (if any) to show as hero
+                scheduled_featured = get_current_featured_media()
+
+                if scheduled_featured:
+                    # Get all other featured videos
+                    other_featured_ids = list(
+                        Media.objects.filter(basic_query, featured=True)
+                        .exclude(pk=scheduled_featured.pk)
+                        .values_list("pk", flat=True)
+                    )
+                    # Combine: scheduled first, then others
+                    all_ids = [scheduled_featured.pk] + other_featured_ids
```

Ruff (0.14.8)

Consider `[scheduled_featured.pk, *other_featured_ids]` instead of concatenation

Replace with `[scheduled_featured.pk, *other_featured_ids]`

(RUF005)